packages/extension-utils/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [21.20.0](https://www.github.com/looker-open-source/sdk-codegen/compare/extension-utils-v0.1.0...extension-utils-v0.1.0) (2021-11-10)
```

Comment:
I'm wondering if this `[21.20.0]` in the changelog is why release-please chose the wrong version. I checked and it isn't the tags.

Also the compare statement didn't make sense, comparing to itself, so I changed it to the all tag of the last release before we added the component.